src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -598,7 +598,7 @@
   "com.minecolonies.coremod.gui.workerhuts.resourcelist": "Required Resources",
 
   "com.minecolonies.coremod.gui.workerhuts.recall": "Recall Worker",
-  "com.minecolonies.coremod.gui.workerhuts.workerassigned": "Assigned Workers:",
+  "com.minecolonies.coremod.gui.workerhuts.workerassigned": "§lAssigned Workers:",
```

Comment:
Can't you modify this on the component level in the code instead of modifying the translation?

Reply:
Technichally yes, but it's more roundabout and we do that with a lot of text already here